hwsc-file-transaction-svc.proto

```diff
@@ -1,37 +1,49 @@
 syntax = "proto3";
 
-package hwscFileTransactionSvc;
+option go_package = "github.com/hwsc-org/hwsc-api-blocks/int/hwsc-file-transaction-svc/file";
+
+package file;
+
+import "lib/authority.proto";
 
 // The file transaction service definition
-service FileTransactionService{
+service FileTransactionService {
     // Gets status for user
-    rpc GetStatus (FileTransactionRequest) returns (FileTransactionResponse){}
+    rpc GetStatus (FileTransactionRequest) returns (FileTransactionResponse) {
+    }
     // Upload files to the storage
-    rpc UploadFile (stream Chunk) returns (FileTransactionResponse){}
+    rpc UploadFile (stream Chunk) returns (FileTransactionResponse) {
+    }
     // Download files from the storage
-    rpc DownloadZippedFiles (FileTransactionRequest) returns (stream Chunk){}
+    rpc DownloadZippedFiles (FileTransactionRequest) returns (stream Chunk) {
+    }
     // Create user folder in the sorage
-    rpc CreateUserFolder(FileTransactionRequest) returns (FileTransactionResponse){}
+    rpc CreateUserFolder (FileTransactionRequest) returns (FileTransactionResponse) {
+    }
 }
 
-message FileTransactionRequest{
-    string message = 1;
-    string name = 2;
-    string uuid = 3;
-    repeated string urls = 4;
+message FileTransactionRequest {
+    lib.Identification identification = 1;
+    string message = 2;
+    string name = 3;
+    string uuid = 4;
+    repeated string urls = 5;
 }
 
-message FileTransactionResponse{
-    oneof status {uint32 code = 1;}
+message FileTransactionResponse {
+    oneof status {
+        uint32 code = 1;
+    }
     string message = 2;
     string url = 3;
 }
 
-message Chunk{
-    oneof test_oneof{
-        bytes buffer = 1;
-        int64 length = 2;
-        string file_name = 3;
-        string uuid = 4;
+message Chunk {
+    oneof is_one_of {
+        lib.Identification identification = 1;
```

Comment:
also here, identification inside the oneof

Reply:
we change the name